lib/index.ts

```diff
@@ -166,16 +166,6 @@ export type MaplibreGeocoderOptions = {
   proximityMinZoom?: number;
   /**
    * A function that specifies how the selected result should be rendered in the search bar. HTML tags in the output string will not be rendered. Defaults to `(item) => item.place_name`.
-   * @example
```

Comment:
This was probably incorrectly placed here, I would suggest to move it above the maplibregeocoder class, where there is a short comment.

Reply:
I think it makes sense to leave this removed, and perhaps there is no need to show getSuggestions and getPlaceById in examples as they are well defined in the types?

Reply:
Type definition and examples are different things... I would prefer to have even more than one example to indicate how to use something. Type definition are important but less helpful in my experience...